### PR TITLE
Isaicp 5084

### PR DESCRIPTION
**Pull request by junkuncz**

### CODE REVIEW

File: web/modules/custom/joinup_search/config/install/facets.facet.type.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ name: Type
 url_alias: type
 weight: 0
 min_count: 1
-show_only_one_result: true
+show_only_one_result: false
```
Comment (claudiu-cristea) [inline web/modules/custom/joinup_search/config/install/facets.facet.type.yml]: We need to test that multiple choices can be selected. Probably you can do that in `tests/features/joinup_search/search.feature`.

File: web/modules/custom/joinup_search/config/install/facets.facet.type.yml [hunk 2]
```diff
@@ -22,6 +22,7 @@ widget:
     soft_limit_settings:
       show_less_label: 'Show less'
       show_more_label: 'Show more'
+    arbitrary_facet_plugin: _none
```
Comment (claudiu-cristea) [inline web/modules/custom/joinup_search/config/install/facets.facet.type.yml]: What is the reason for this change. This made tests to fail.

Comment (junkuncz) [reply web/modules/custom/joinup_search/config/install/facets.facet.type.yml]: Looks like config export put this line automatically during the export. Don't think that it relates to my current feature so will remove it.

File: web/modules/custom/joinup_search/config/install/facets.facet.type.yml (hunk 1 above)
Comment (junkuncz) [reply web/modules/custom/joinup_search/config/install/facets.facet.type.yml]: I'm on it!

Comment (junkuncz) [reply web/modules/custom/joinup_search/config/install/facets.facet.type.yml]: Test added to multiple choices and broken tests also have been fixed.
